esp/public/media/scripts/onsite/ajax_status.js

```diff
@@ -201,9 +248,10 @@ function print_schedule()
     result = $j.ajax({
         url: printing_url,
         data: student_data,
-        async: false
+        success: function (data, text_status, jqxhr) {
+            add_message(data.message);
```

Comment:
Oh, that's right, I forgot about that.  Seems like it wouldn't hurt to declare it explicitly if we can.